README.md

```diff
@@ -12,9 +12,9 @@ Demos
 
 Installation Instructions
 -------------------------
-Loris is supported on Python 2.7.x and Python 3.4+. If you need to use Python 2.6, Loris v2.1.0 is the last version to support it.
+Loris is supported on Python 3.5+. If you need to use Python 2.7, Loris v2.3.3 is the last version to support it.
```

Comment:
“If you need to use Python 2.7 *or Python 3.4*”?

Reply:
done, thanks.